ClusterTest/src/main/scala/net/snowflake/spark/snowflake/TaskContext.scala

```diff
@@ -0,0 +1,8 @@
+package net.snowflake.spark.snowflake
+
+private[snowflake] case class TaskContext(var testName: String) {
```

Comment:
What about using TestContext or TestSuiteContext to distinguish with Spark TaskContext?

Reply:
Changed.